fleetspeak/src/client/clienttestutils/clienttestutils_windows.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build windows
+
+package clienttestutils
+
+import (
+	"fmt"
+
+	"github.com/google/fleetspeak/fleetspeak/src/windows/regutil"
+
+	fspb "github.com/google/fleetspeak/fleetspeak/src/common/proto/fleetspeak"
+)
+
+func writeServiceConfigImpl(content []byte, dirpath, filename string, cfg *fspb.SignedClientServiceConfig) {
```

Comment:
This compiles?

Reply:
Interesting. Looks like I forgot to push

Reply:
git push -f stopped working for me for whatever reason. I'll investigate some more.

Reply:
Done.